aces/Loris
# [new_profile] Auto-select only site option

## Brief summary of changes

Resolves issue https://github.com/aces/Loris/issues/5380

#### Testing instructions (if applicable)

(*make sure to run `make dev` when you switch branches*)
1. On main branch, on a user account affiliated with multiple sites, go to New Profile module and create a new candidate. You are required to choose a site on the form.
2. Repeat the above step but on a user account affiliated with only one site. Notice that the 'Site' field does not appear and on successful creation of the candidate, the candidate is created for the correct site.
3. On this PR branch, repeat the above steps. With a multiple-site user account, there should be no changes. With a single-site user account, notice that the 'Site' field appears and that the only site option is auto-selected. You should get no errors creating the new candidate.
4. Finally, test a couple of different select elements, both with multiple options and single options, in Loris to ensure that no unexpected behaviours appear. (select elements within module menu filters should not be affected because autoSelect is set to false there)

#### Link(s) to related issue(s)

* Resolves https://github.com/aces/Loris/issues/5380
